user-workspace/src/App.tsx

```
@@ -22,77 +23,57 @@ const config = {
-      const verificationResult = await reclaimVerification.startVerificationFromUrl(verificationUrl);
+      let verificationResult: ReclaimVerification.Response;
+      switch (verificationMethod) {
+        case 'provider':
+          verificationResult = await reclaimVerification.startVerification({
+            appId: config.REACT_APP_RECLAIM_APP_ID ?? '',
+            secret: config.REACT_APP_RECLAIM_APP_SECRET ?? '',
+            providerId: inputText,
+          });
+          break;
+        case 'json':
+          verificationResult = await reclaimVerification.startVerificationFromJson(JSON.parse(inputText));
```

ESLint

[error] Replace `·await·reclaimVerification.startVerificationFromJson(JSON.parse(inputText)` with `⏎············await·reclaimVerification.startVerificationFromJson(⏎··············JSON.parse(inputText)⏎············`

(prettier/prettier)